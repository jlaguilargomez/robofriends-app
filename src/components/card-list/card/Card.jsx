import React from 'react';
// import * from './Card.css'

const Card = ({ id, name, email }) => {

  return (
    <div className="tc bg-light-green dib br3 pa3 ma2 grow shadow-5">
      <img src={`https://robohash.org/ass${id}`} alt="robot imagen"></img>
      <div>
        <h2>{name}</h2>
        <p>{email}</p>
      </div>
    </div>
  );
};

export default Card;
