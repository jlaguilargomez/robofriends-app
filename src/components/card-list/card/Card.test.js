import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card';

describe('CardComponent', () => {
  const app = shallow(<Card></Card>);


  it('expect to render Card component', () => {
    // 'shallow' nos permite testear componentes sin depender de sus sub-componentes (poco-profundo)
    expect(app).toMatchSnapshot();
  });
})
