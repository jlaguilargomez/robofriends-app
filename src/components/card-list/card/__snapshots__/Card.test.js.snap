// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardComponent expect to render Card component 1`] = `
<div
  className="tc bg-light-green dib br3 pa3 ma2 grow shadow-5"
>
  <img
    alt="robot imagen"
    src="https://robohash.org/assundefined"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
